_attributes/common-attributes.adoc

```diff
@@ -318,4 +318,10 @@ endif::[]
 :cap-oci-short: Cluster API Provider OCI
 // Cluster API Provider VMware vSphere
 :cap-vsphere-first: Cluster API Provider VMware vSphere
-:cap-vsphere-short: Cluster API Provider vSphere
\ No newline at end of file
+:cap-vsphere-short: Cluster API Provider vSphere
+
+// Hosted control planes related attributes
+:mce-first: multicluster engine for Kubernetes operator (MCE)
+:mce-short: MCE
+:hcp-first: Hosted control planes
```

Comment:
> This term with capital H is used only in the beginning of the sentence. Otherwise "H" is not capitalized.

I would suggest using something other than `-first`, since that convention is typically for things like "Amazon Web Services (AWS)" which are written out plus their abbreviation in parens on first appearance in a page.

Not sure what is a good alternative, maybe `-sentence` or `-initial` or `-capital`?

```suggestion
:hcp-first: Hosted control planes
```

Reply:
That is correct!
Thanks for catching it. 
Updated to :hcp-capital: